requirements/requirements.linux-kubuntu.txt

```
@@ -10,7 +10,7 @@ bleach==6.0.0
 cffi==1.15.1
 comm==0.1.3
 contourpy==1.0.7
-cryptography==40.0.2
+cryptography==41.0.0
```

use latest version instead
```suggestion
cryptography==41.0.1
```